bin/common/install_tools.sh

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env bash
+set -o errexit -o nounset
+
+# Get version information and set destination dirs
+. "$(dirname "$0")/versions.sh"
+
+# Installs tools needed to build and run HCF
+SCF_BIN_DIR="${SCF_BIN_DIR:-/home/vagrant/bin}"
+SCF_TOOLS_DIR="${SCF_TOOLS_DIR:-/home/vagrant/tools}"
+cf_url="${cf_url:-https://cli.run.pivotal.io/stable?release=linux64-binary&version=${CFCLI_VERSION}&source=github-rel}"
+kubectl_url="${kubectl_url:-https://storage.googleapis.com/kubernetes-release/release/v${KUBECTL_VERSION}/bin/linux/amd64/kubectl}"
+k_url="${k_url:-https://github.com/aarondl/kctl/releases/download/v${K_VERSION}/kctl-linux-amd64}"
+kk_url="${kk_url:-https://gist.githubusercontent.com/jandubois/${KK_VERSION}/raw/}"
```

Comment:
This seems to work for me: https://gist.githubusercontent.com/jandubois/40a5b3756cf4bcbed940e6156272c0af/raw

`KK_VERSION` is set in https://github.com/SUSE/scf/pull/1021/files#diff-e482966f3ca1c1f6ed808625edfe39c2R14.. 

Ah! You mean it should now be `576a42386770423ced46ab4ae9955bee59b0d4dd`?